pkg/artifacts/commenter/commenter.go

```diff
@@ -213,6 +216,7 @@ func (c *Commenter) generatePRComments(ctx context.Context) ([]pullRequest, erro
 	}
 
 	var pullRequests []pullRequest
+	 updatedRunIDs := map[string]struct{}{}
```

Comment:
prevents overflow of run ID. It only keeps active runIDs. expeired workflow or closed PRs are not included.